Compile error Kodi 22 Alpha 1 on Gentoo

Running a RPI5 with gentoo. I am trying to compile but it throws a missing gcc compiler. I saw there was another bug that said it was fixed but I am still seeing the error.

`checking ABI=32
checking compiler gcc -O2 -pedantic -fomit-frame-pointer ... no, long long reliability test 1
checking whether cc is gcc... yes
checking compiler cc -O2 -pedantic -fomit-frame-pointer ... no, long long reliability test 1
configure: error: could not find a working compiler, see config.log for details
make[5]: *** [CMakeFiles/gmp.dir/build.make:92: gmp-prefix/src/gmp-stamp/gmp-configure] Error 1
make[4]: *** [CMakeFiles/Makefile2:87: CMakeFiles/gmp.dir/all] Error 2
make[3]: *** [Makefile:136: all] Error 2
make[2]: *** [CMakeFiles/gmp.dir/build.make:86: build/gmp/src/gmp-stamp/gmp-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:341: CMakeFiles/gmp.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
`
from config.log
`conftest.c: In function 'f':
conftest.c:12:48: error: too many arguments to function 'g'; expected 0, have 6
   12 | for(i=0;i<1;i++){if(e(got,got,9,d[i].n)==0)h();g(i,d[i].src,d[i].n,got,d[i].want,9);if(d[i].n)h();}}
      |                                                ^ ~
conftest.c:7:6: note: declared here
    7 | void g(){}
      |      ^
configure:6588: $? = 1`

--- a/depends/common/gmp/CMakeLists.txt
+++ b/depends/common/gmp/CMakeLists.txt
@@ -53,6 +53,7 @@ else()
 
   externalproject_add(gmp
                       SOURCE_DIR ${CMAKE_SOURCE_DIR}
+                      PATCH_COMMAND autoreconf -vif
                       CONFIGURE_COMMAND <SOURCE_DIR>/configure
                         --prefix=${CMAKE_INSTALL_PREFIX}
                         --disable-assembly

